egs2/TEMPLATE/enh_asr1/scripts/audio/format_wav_scp.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
This file seems to be a copy of [egs2/TEMPLATE/asr1/scripts/audio/format_wav_scp.sh](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/scripts/audio/format_wav_scp.sh).
It is better to make a symbolic link instead, like [egs2/TEMPLATE/enh1/scripts/audio](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/enh1/scripts/audio).

Reply:
Thanks. Have addressed this and the following similar comments.